lib/WebService/Stripe.pm

```diff
@@ -60,14 +60,21 @@ method create_card(HashRef $data, :$customer_id!, :$headers) {
         "/v1/customers/$customer_id/cards", $data, headers => $headers );
 }
 
-method get_charge(Str $id, :$headers) {
-    return $self->get( "/v1/charges/$id", {}, headers => $headers );
+method get_charge(HashRef|Str $charge, :$expand, :$headers) {
+    my $path = "/v1/charges/".(ref $charge ? $charge->{id} : $charge);
+    $path .= '?' . join '&', map { "expand[]=$_" } @$expand if $expand;
```

Comment:
Should not construct query string by hand. Pass the query params as a hashref as the second argument to `->get`.